kitchen/wazuh-puppet/Dockerfile

```diff
@@ -0,0 +1,79 @@
+ARG DEBIAN_FRONTEND=noninteractive
+
+FROM ubuntu:18.04
+
+# Global required packages
+RUN apt-get update -qq -y && \
+    apt-get upgrade -qq -y && \
+    apt-get install -qq -y \
+      build-essential \
+      apt-transport-https \
+      ca-certificates \
+      curl \
+      software-properties-common \
+      dirmngr \
+      libreadline-dev \
+      libssl-dev \
+      lsb-release \
+      puppet \
+      rbenv \
+      ruby \
+      ruby-dev \
+      rubygems \
+      sudo \
+      systemd \
+      systemd-sysv \
+      wget \
+      zlib1g-dev
+
+# docker-ce related installation tasks.
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add - && \
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable" && \
+    apt update -qq -y && \
+    apt-cache policy docker-ce && \
+    apt install docker-ce -y
+
+
+# Required gems.
+RUN gem install rake \
+      --no-document \
+      --quiet && \
+    gem install bundler \
+      --no-document \
+      --quiet && \
+    gem install kitchen-ansible \
+      --no-document \
+      --quiet && \
+    gem install kitchen-docker \
+      --no-document \
+      --quiet && \
+    gem install kitchen-docker_cli \
+      --no-document \
+      --quiet && \
+    gem install kitchen-puppet \
+      --no-document \
+      --quiet && \
+    gem install kitchen-verifier-serverspec \
+      --no-document \
+      --quiet && \
+    gem install librarian-puppet \
+      --no-document \
+      --quiet && \
+    gem install test-kitchen \
+      --no-document \
+      --quiet && \
+    gem install kitchen-goss \
+      --no-document \
+      --quiet
+
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python3 get-pip.py
+
+RUN pip install pytest && \
+    pip install testinfra && \
+    pip install paramiko
+
+# Adding systemd compatibility.
+VOLUME [ "/sys/fs/cgroup", "/run", "/run/lock" ]
```

Comment:
Please check if the container runs in privileged mode (https://stackoverflow.com/questions/32144575/how-to-know-if-a-docker-container-is-running-in-privileged-mode) or only mounts the cgroup

Reply:
Hi @jm404!

As checked, the container does not run in `privileged` mode.

```bash
vagrant@puppet:~/wazuh-qa/kitchen/wazuh-puppet$ docker inspect --format='{{.HostConfig.Privileged}}' b00a3a310750a3017d478cf40df098be42ba61dcedb170a8446c7959a360ad9b
false
```

Kr,

Rshad